server/sonar-server/src/main/java/org/sonar/server/rule/RuleCreator.java

```diff
@@ -39,6 +39,7 @@
 import org.sonar.db.rule.RuleDefinitionDto;
 import org.sonar.db.rule.RuleDto;
 import org.sonar.db.rule.RuleDto.Format;
+import org.sonar.db.rule.RuleDto.Scope;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.db.rule.RuleDto.Scope'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fdm%2Frule_scope&issues=AWFHHYE2KuAw5oyFANws&open=AWFHHYE2KuAw5oyFANws)

[ISSUE_KEY]: AWFHHYE2KuAw5oyFANws